draft-regext-rdap-extensions.md

```diff
@@ -477,12 +506,16 @@ identifier pattern, that example could be written as:
       }
     }
 
-Usage of a bare extension identifier contravenes the guidance in
-[@!RFC9083].  This document updates [@!RFC9083] to explicitly allow
-this pattern.
-
 Along similar lines, an extension may define a single new object
-class, and use the extension's identifier as the object class name.
+class, and use the extension's identifier as the object class name
+(see (#object_classes_in_extensions)).
+A> issue #52
+
+Usage of a bare extension identifier conflicts with the guidance in
+[@!RFC9083, section 2.1].  This document clarifies [@!RFC9083, Section 2.1] to explicitly allow
+this pattern but advises extension authors to use this pattern when only one
```

Comment:
For now, change "advises" to something more normative? Like "MUST NOT use them unless only one...".